File: CS/RegisterFromLogonFormSolution.Module.Win/WinModule.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;

using DevExpress.ExpressApp;
using WinWebSolution.Module.Controllers;

namespace RegisterFromLogonFormSolution.Module.Win {
    [ToolboxItemFilter("Xaf.Platform.Win")]
    public sealed partial class RegisterFromLogonFormSolutionWindowsFormsModule : ModuleBase {
        public RegisterFromLogonFormSolutionWindowsFormsModule() {
            InitializeComponent();
        }
        public override void Setup(XafApplication application) {
            base.Setup(application);
            application.CreateCustomLogonWindowControllers += application_CreateCustomLogonWindowControllers;
        }
        private void application_CreateCustomLogonWindowControllers(object sender, CreateCustomLogonWindowControllersEventArgs e) {
            XafApplication app = (XafApplication)sender;
            e.Controllers.Add(app.CreateController<LogonActionParametersController>());
        }
    }
}

File: VB/RegisterFromLogonFormSolution.Module.Win/WinModule.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Imports System.ComponentModel

Imports DevExpress.ExpressApp
Imports WinWebSolution.Module.Controllers

Namespace RegisterFromLogonFormSolution.Module.Win
	<ToolboxItemFilter("Xaf.Platform.Win")> _
	Public NotInheritable Partial Class RegisterFromLogonFormSolutionWindowsFormsModule
		Inherits ModuleBase
		Public Sub New()
			InitializeComponent()
		End Sub
		Public Overrides Overloads Sub Setup(ByVal application As XafApplication)
			MyBase.Setup(application)
			AddHandler application.CreateCustomLogonWindowControllers, AddressOf application_CreateCustomLogonWindowControllers
		End Sub
		Private Sub application_CreateCustomLogonWindowControllers(ByVal sender As Object, ByVal e As CreateCustomLogonWindowControllersEventArgs)
			Dim app As XafApplication = CType(sender, XafApplication)
			e.Controllers.Add(app.CreateController(Of LogonActionParametersController)())
		End Sub
	End Class
End Namespace
